### DISCUSSION TITLE
Porduino update to 2.5.14 breaks web client

### DISCUSSION THREAD

**Discussion by TheSmashy** [meshtastic/firmware#5512] [Troubleshooting]
I updated my Linux native node / Portduino to 2.5.14 via the arm64 deb package, and everything looks good via the python cli (even sent a test message) but I cannot access the web client via the node's IP address.  I just get a white page.  I have tried different browsers and devices on my LAN, same results, checked my config.yaml, no joy.  Any ideas?

**Comment by eusef** [comment]
i had the same problem, unzip the zipped files. there is a fix for this but it is not out yet.  

They can be found in - /usr/share/doc/meshtasticd/web 

There are a few folders in there

**Comment by fifieldt** [comment]
This was a bug which is now fixed :)
